unsplashi/listdownloader.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+def getList(url):
+	client_id = "a235801b84ae3057163a384548acbb74f82dfc72ee2e8e9580637271ea1a7858"
```

Comment:
@chiragbablani0 , 
- let's make a `config.sample.py` file, and add `config.py` to .gitignore, in config.sample.py file add this client-id as a variable.
- put a dummy value there.

Reply:
@chiragbablani0 in this file, import 'client_id' from config.py and use the variable here